doc/source/io.rst

```diff
@@ -282,16 +285,16 @@ float_precision : string, default None
   Specifies which converter the C engine should use for floating-point values.
   The options are ``None`` for the ordinary converter, ``high`` for the
   high-precision converter, and ``round_trip`` for the round-trip converter.
-lineterminator : str (length 1), default ``None``
```

Comment:
so anything you are actually deprecating (which is pretty much everything that is changed), needs to have a either a separate entry for a doc-string (like you did for ``skip_footer``), or prob better, show the doc-string for ``skip_footer``  then at the end put the DEPRECATION OF ``skipfooter`` or somesuch. 

Futher need a BIG BIG warning in v0.19.0 (use ``::warning``).